rules/optimize-google-my-business-profile/rule.md

```diff
@@ -1,44 +1,58 @@
-2. **Address** – correct street number and suite number, if needed
-3. **Phone** – is number the same one you have on your website?
-4. **Hours** - add the correct ones.
-   * Public holidays - update them at the beginning of every year. Can you imagine a client showing up to a closed business?
-5. **Business description and category** – add this information to help attract potential customers
-6. **Customer reviews** – whether you get good or bad reviews, make sure you reply to them.
-   * If there was a problem, try to understand the situation and help them.
+1. **Business name** – Make sure it's [consistent](/the-value-of-consistency)) with your website and social media
+2. **Address** – Have the accurate [street and suite number in the right format](/address-formatting)
+3. **Phone** – Have the accurate [phone number in the right format](/right-format-to-show-phone-numbers)
+4. **Hours** - Include the accurate business hours
+   * Public holidays - Update them at the beginning of every year
+5. **Description and category** – Add informative information to help attract potential customers
+6. **Customer reviews** – Make sure you [reply to good or bad reviews](/monitor-external-reviews/#respond-to-feedback-appropriately)
+   * If there was a problem, try to understand the situation and help them
    * If you received a glowing review, say thanks!
-7. **Products/services** – in this description field, use SEO keywords that people usually use to look for the product/service. This will make your profile more ‘[Googable](https://www.urbandictionary.com/define.php?term=googable)’.
-8. **Photos** - upload cool and real photos of your business.
+7. **Products/services** – Use keywords that people usually use to look for your products and services
+8. **Photos** - Upload cool and real photos of your business
    * Front and inside of your business
    * Your team
    * Events
-   * Specific shots e.g. your menu if you own a restaurant.
-9. **Posting feature** – if you want to share a new service you are offering or hosting an event, you can use this feature to communicate it to your customers.
-10. **Mobile app** - add your business to your Google Maps mobile app, so you can make edits when needed
+   * Logo
+   * Specific shots e.g. your menu if you own a restaurant
+9. **Posting feature** – Share a new service you are offering or an event you are hosting
 
-  * Do this by adding your personal Gmail address as a Manager account:
+::: info
+If you ever need to change your phone number or move to a new address, **prioritize** updating your Google My Business listing.
+:::
+
+## Managing the business profile
+
+You must verify your business before you can edit information about it. After verification of the business and approval of the edits, these changes can be published on Google Search and Maps.
+
+You may want to add a marketing member Gmail as a Manager account to avoid signing in with the company account for authentication.
```

Comment:
```suggestion
You may want to add a marketing member's Gmail account as a Manager to avoid signing in with the company account for authentication.
```